public/documentation/webcl.md

```diff
@@ -0,0 +1,284 @@
+# Overview
+
+[WebCL](https://www.khronos.org/webcl/) defines a JavaScript binding to
+[the Khronos OpenCL standard](https://www.khronos.org/opencl/) for
+heterogeneous parallel computing. WebCL enables web applications to
+harness GPU and multi-core CPU parallel processing and achieve
+significant acceleration. The WebCL 1.0 specification was released by
+Khronos Group in 2014, which is based on OpenCL 1.1 (Embedded Profile).
+
+**WebCL does not make Facebook run faster on your browser. According to
+Amdahl's law, the speedup of a program using multiple processors in
+parallel computing is limited by the serial part of the program.**
+
+WebCL has been enabled by default on Android since Crosswalk 14.
+Currently, the feature requires OpenCL runtime library to be installed.
+Some OEM ship the OpenCL runtime library by default in their products,
+and the library is typically named as `libOpenCL.so`, `libOpenCL.so.1`
+or `libPVROCL.so` in `/system/lib/` or `/system/vendor/lib/`. If you
+cannot find any of these libraries, then unfortunately WebCL will not
+work on your device, and you probably want to manually install the
+OpenCL runtime library.
```

Comment:
is there an easier way to check for OpenCL support in an Android device? I remember seeing at least one app in the Play Store. If so, it could be mentioned here